Sources/SourceKitLSP/Swift/SyntaxHighlightingToken.swift

```diff
@@ -47,146 +47,8 @@ public struct SyntaxHighlightingToken: Hashable {
-      case .enumMember: return "enumMember"
-      case .event: return "event"
-      case .function: return "function"
-      case .method: return "method"
-      case .macro: return "macro"
-      case .keyword: return "keyword"
-      case .modifier: return "modifier"
-      case .comment: return "comment"
-      case .string: return "string"
-      case .number: return "number"
-      case .regexp: return "regexp"
-      case .operator: return "operator"
-      case .decorator: return "decorator"
-      case .identifier: return "identifier"
-      }
-    }
-
-    /// **Public for testing**
-    public var _lspName: String {
-      lspName
-    }
-  }
-
-  /// Additional metadata about a token.
-  ///
-  /// Similar to `Kind`, the raw values do not actually have
-  /// to be stable, do note however that the bit indices should
-  /// be numbered starting at 0 and that the ordering should
-  /// correspond to `allModifiers`.
-  public struct Modifiers: OptionSet, Hashable {
-    public static let declaration = Self(rawValue: 1 << 0)
-    public static let definition = Self(rawValue: 1 << 1)
-    public static let readonly = Self(rawValue: 1 << 2)
-    public static let `static` = Self(rawValue: 1 << 3)
-    public static let deprecated = Self(rawValue: 1 << 4)
-    public static let abstract = Self(rawValue: 1 << 5)
-    public static let async = Self(rawValue: 1 << 6)
-    public static let modification = Self(rawValue: 1 << 7)
-    public static let documentation = Self(rawValue: 1 << 8)
-    public static let defaultLibrary = Self(rawValue: 1 << 9)
-
-    /// All available modifiers, in ascending order of the bit index
-    /// they are represented with (starting at the rightmost bit).
-    public static let allModifiers: [Self] = [
-      .declaration,
-      .definition,
-      .readonly,
-      .static,
-      .deprecated,
-      .abstract,
-      .async,
-      .modification,
-      .documentation,
-      .defaultLibrary,
-    ]
-
-    public let rawValue: UInt32
-
-    /// The name of the modifier used by LSP, if this
-    /// is a single modifier. Note that every modifier
-    /// in `allModifiers` must have an associated `lspName`.
-    var lspName: String? {
-      switch self {
-      case .declaration: return "declaration"
-      case .definition: return "definition"
-      case .readonly: return "readonly"
-      case .static: return "static"
-      case .deprecated: return "deprecated"
-      case .abstract: return "abstract"
-      case .async: return "async"
-      case .modification: return "modification"
-      case .documentation: return "documentation"
-      case .defaultLibrary: return "defaultLibrary"
-      default: return nil
-      }
-    }
-
-    /// **Public for testing**
-    public var _lspName: String? {
-      lspName
-    }
-
-    public init(rawValue: UInt32) {
-      self.rawValue = rawValue
-    }
-  }
+  public typealias Kind = SemanticTokenTypes
+  public typealias Modifiers = SemanticTokenModifiers
```

Comment:
Do we need these `typealias` declarations or can we just use `SemanticTokenTypes` and `SemanticTokenModifiers`?

Reply:
no. changed.